automobile/scenario-creation-tutorial.md

```diff
@@ -0,0 +1,78 @@
+## Creation of a real world scenario
+
+This small tutorial will explain step by step how to create a real scenario inspired from a real world map and then add traffic using SUMO.
```

Comment:
will explain &rarr; explains
step by step &rarr; step-by-step
I would remove "real" in "real scenario".